sklearn/metrics/regression.py

```diff
@@ -90,7 +90,10 @@ def _check_reg_targets(y_true, y_pred, multioutput):
     n_outputs = y_true.shape[1]
     multioutput_options = (None, 'raw_values', 'uniform_average',
                            'variance_weighted')
-    if multioutput not in multioutput_options:
+    if isinstance(multioutput, string_types) and multioutput not in multioutput_options:
```

Comment:
I will refine the if else to make sure we account for the correct multioutput check there.